storage_backup_schedule.go

```diff
@@ -0,0 +1,207 @@
+package gsclient
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"path"
+)
+
+type StorageBackupScheduleList struct {
+	List map[string]StorageBackupScheduleProperties `json:"schedule_storage_backups"`
+}
+
+type StorageBackupSchedule struct {
+	Properties StorageBackupScheduleProperties `json:"schedule_storage_backup"`
+}
+
+type StorageBackupScheduleProperties struct {
+	//Defines the date and time of the last object change.
+	ChangeTime GSTime `json:"change_time"`
+
+	//Defines the date and time the object was initially created.
+	CreateTime GSTime `json:"create_time"`
```

Comment:
Just a question, I think I understand by now why this type is used. Why call it GSTime, though? Why not just DateTime or ISODateTime or so?

Reply:
If you check the `gsTimeLayout` constant in `types.go` file, you will see the reason. `GSTime` is actually `time.Time` and since gridscale use different time date layout, it is not possible for `go` to unmarshall the reponse's time to `time.Time`, we have to custom `UnmarshallJSON` method.

Reply:
Really? Isn't this just [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) and would be just something like that
```
sometime.Format(time.RFC3339)
```
when marshaling / un-marshaling? Don't want to be bike-shedding here, though. Code is fine, just don't understand the naming of GSTime.

Reply:
Yes. I have tried and it didn't work with RFC3339. The format of time when receiving a response is like this 
![image](https://user-images.githubusercontent.com/9968099/91866861-43bbd980-ec73-11ea-8f35-f546a852a347.png)

Reply:
AH, you're right. Didn't see that. Thanks for clarifying.